src/config/gobjects.rs

```diff
@@ -417,6 +420,10 @@ fn parse_object(
         .lookup("clear_function_expression")
         .and_then(Value::as_str)
         .map(ToOwned::to_owned);
+    let default_impl_value = toml_object
+        .lookup("default_impl_value")
```

Comment:
```suggestion
        .lookup("default_value")
```

Seems more descriptive

Reply:
default_value looks good to me as well